scimath/Mathematics/ClassicalStatistics.h

```diff
@@ -38,6 +38,9 @@
 #include <vector>
 #include <utility>
 
+// because the template signature has become unwieldy
+#define _TPARMS AccumType, DataIterator, MaskIterator, WeightsIterator
```

Comment:
hm underscore followed by uppercase letter is reserved by the c++ standard for any use (17.6.4.3.2)
I a header it should at least be `#undef` at the end or have a `CASA? string in it